res/values/config.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="url" translatable="false">https://seamlessupdate.app/</string>
+    <string name="channel_default" translatable="false">stable</string>
+    <string name="network_type_default" translatable="false">1</string>
+    <string name="battery_not_low_default" translatable="false">false</string>
+    <string name="idle_reboot_default" translatable="false">false</string>
```

Comment:
I left these defaults as they were, but you might consider changing them after merging.